examples/step-1/doc/intro.dox

```diff
@@ -128,6 +128,13 @@ command line, you may be interested in watching lectures 2.9 and 2.91.
 (@dealiiVideoLectureSeeAlso{2.9,2.91}) These give overviews over the command
 line and on what happens when compiling programs, respectively.
 
+Note that deal.II is actively developed, and in the course of this
+development we occasionally rename or deprecate functions or classes
+that are still referenced in these video lectures.  For
+example, step-1 code shown in the video lecture 5 has a class
```

Comment:
I know -- neither am I, but at least my native language uses "the" and "a" in a broadly similar way to English :-)